### PR TITLE
Docs: Share panel image preview

### LINKED ISSUE #97814: ShareLink: Render panel - Update docs
The render panel is now performed inside the Share link drawer, instead of opening a new tab. 
We need to change [these steps](https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/#use-direct-link) to visualize the changes and the new wordings as well

### PR DESCRIPTION
**Pull request by imatwawana**
Update docs for new panel image share options and image rendering

### CODE REVIEW

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 1]
```diff
@@ -239,10 +239,15 @@ To share a personalized, direct link to your panel within your organization, fol
    - **Lock time range** - Change the current relative time range to an absolute time range. This option is enabled by default.
    - **Shorten link** - Shorten the panel link. This option is disabled by default.
 1. Select the theme for the dashboard. Choose from **Current**, **Dark**, or **Light**.
-1. Do one or both of the following:
-   - Click **Copy link**.
-   - Click **Render image**, which [renders the panel as a PNG image](ref:image-rendering).
-1. Send the copied link or image URL to a Grafana user with authorization to view it.
+1. Click **Copy link**.
+1. Send the copied link to a Grafana user with authorization to view it.
+1. (Optional) To [generate an image of the panel as a PNG file](ref:image-rendering), customize the image settings:
```
Comment (juanicabanas) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:244]: I think it worth mentioning that there are maximums in the image renderer configuration for width and height as well as scale factor https://grafana.com/docs/grafana/latest/setup-grafana/image-rendering/#viewport-height

File: docs/sources/dashboards/share-dashboards-panels/_index.md
Comment (juanicabanas) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:1]: We should also change the `Query string parameters for server-side rendered images` section


When you click **Generate image** in the panel link settings, Grafana generates a PNG image of the panel with the following default parameters:

| Parameter | Description                                                                                                                    |
| --------- | ------------------------------------------------------------------------------------------------------------------------------ |
| width     | Width in pixels. Default is 1000.                                                                                               |
| height    | Height in pixels. Default is 500.                                                                                              |
| tz        | Timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC.                              |
| timeout   | Number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.             |
| scale     | Numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). |

You can also update these parameters in the [image rendering configuration](ref:image-rendering-config).
The following example shows a link to a server-side rendered PNG:
```bash
https://play.grafana.org/render/d-solo/ktMs4D6Mk?from=2024-09-03T11:55:44.442Z&to=2024-09-03T17:55:44.442Z&panelId=panel-13&__feature.dashboardSceneSolo&width=1000&height=500&tz=UTC
```

Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:1]: Good catch! See [here](https://github.com/grafana/grafana/pull/98039/files#diff-3a2275957644c70551aa45e79e8cb5862f0123ce8bf2f1191850f28dec098fa7R255).

**Review by juanicabanas — APPROVED** [review]
LGTM. Great work

**Pull request merged**